### PR TITLE
Esperanto: Translation of new items

### PR DESCRIPTION
**Pull request by Keksoj**
Almost all new items translated here. Would new an esperanto-speaking peer review but it'll do for now.

### CODE REVIEW

File: translations/monero-core_eo.ts [hunk 1]
```diff
@@ -784,107 +784,107 @@
...
         <source>Help</source>
-        <translation type="unfinished"></translation>
+        <translation>Helpo</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="206"/>
         <location filename="../pages/Receive.qml" line="435"/>
         <source>&lt;p&gt;This QR code includes the address you selected above andthe amount you entered below. Share it with others (right-click-&gt;Save) so they can more easily send you exact amounts.&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;Tiu QR-kodo inkluzivas la adreson kiun vi elektis ĉi-supre, kaj la kvanton kiun vi entajpis sube. Donu ĝin al aliuloj (dekstroklako-&g;konservi) tiel ke ili pli facile povu sendi al vi ekzaktajn kvantojn.&lt;/p&gt;</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: `(dekstroklako-&gt;konservi)` <- the entity here must be `&gt;` not `&g;`

File: translations/monero-core_eo.ts [hunk 2]
```diff
@@ -743,20 +743,20 @@
     <message>
         <location filename="../components/ProgressBar.qml" line="38"/>
         <source>%1 blocks remaining: </source>
-        <translation type="unfinished"></translation>
+        <translation>Restas %1 blokoj</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: Please add `:` + whitespace at the and of the string

File: translations/monero-core_eo.ts [hunk 3]
```diff
@@ -1048,12 +1048,12 @@
...
         <location filename="../pages/Settings.qml" line="680"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;a href=&apos;#&apos;&gt; (Klaki por ŝanĝi)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="682"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Agordi novan restarig-altecon</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: missing `:`

File: translations/monero-core_eo.ts [hunk 4]
```diff
@@ -1063,22 +1063,22 @@
...
         <location filename="../pages/Settings.qml" line="657"/>
         <source>Wallet name: </source>
-        <translation type="unfinished"></translation>
+        <translation>Nomo de la monujo:</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="671"/>
         <source>Wallet creation height: </source>
-        <translation type="unfinished"></translation>
+        <translation>Alteco dum la monujkreado</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: missing `:` + whitespace

File: translations/monero-core_eo.ts [hunk 5]
```diff
@@ -1090,17 +1090,24 @@ The following information will be deleted
...
         <location filename="../pages/Settings.qml" line="717"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>La entajpita alteco de la restarigo ne validas. Devas esti nombro.</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="733"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Vojo al monujhistorio</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: missing `:` + whitespace

File: translations/monero-core_eo.ts [hunk 6]
```diff
@@ -1214,100 +1221,112 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="135"/>
         <source>You should only have to load a file when you want to refresh the list. Manual adding/removing is possible if needed.</source>
-        <translation type="unfinished"></translation>
+        <translation>Vi devus ŝargi dosieron nur se vi volas refreŝigi la liston. Se necese, eblas mane aldoni/depreni.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="147"/>
         <source>Please choose a file to load blackballed outputs from</source>
-        <translation type="unfinished"></translation>
+        <translation>Bonvolu elekti dosieron el kie ŝargiĝos la malakceptitaj eligoj.</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: remove `.` at the end of the string

File: translations/monero-core_eo.ts [hunk 7]
```diff
@@ -1322,47 +1341,47 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/SharedRingDB.qml" line="392"/>
         <source>Relative</source>
-        <translation type="unfinished"></translation>
+        <translation>Relativa</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="408"/>
         <source>Segregation height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Blokalteco de la disforkiĝo</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: missing `:`

File: translations/monero-core_eo.ts [hunk 8]
```diff
@@ -2616,64 +2622,57 @@ Fee: </source>
...
+        <source>Description: </source>
+        <translation>Priskribo:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="632"/>
-        <source>
-Spending address index: </source>
-        <translation type="unfinished"></translation>
+        <source>Spending address index: </source>
+        <translation>Indekso de elspezanta adreso</translation>
```
Comment (erciccione) [inline translations/monero-core_eo.ts]: missing `:` + whitespace

**Comment by Keksoj** [comment]
All right, I did the minor fixes you recommended, @erciccione , hope I didn't screw up but it seems all right. 
@dnaleor : Dankon por tiu komplimento ! I changed "dekstroklako" to "dekstra klako". I used "duoble klaku" for "double click" in the imperative, it suits better. Yet I have no clue why the english version says "double tap", maybe it's intended for touchscreens ?

**Pull request merged**